machinelearning/7.0.x/drools-learner-examples/pom.xml

```diff
@@ -1,46 +1,61 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
some crazy formatting here. We just indent xml by 2 chars. Maybe you could undo this?